simulation/halsim_ws_client/src/main/native/include/HALSimWS.h

```diff
@@ -67,6 +71,9 @@ class HALSimWS : public std::enable_shared_from_this<HALSimWS> {
   std::string m_host;
   std::string m_uri;
   int m_port;
+
+  bool m_useMsgFiltering;
+  std::set<std::string_view> m_msgFilters;
```

Comment:
It seems like this would be unsafe--what's the backing storage for the string_view?  Maybe consider using `wpi::StringMap<bool>`?